typings/index.d.ts

```diff
@@ -115,12 +115,21 @@ declare module 'klasa' {
 		// Klasa Command Events
 		public on(event: 'commandError', listener: (msg: CommandMessage, command: Command, params: any[], error: Error) => void): this;
 		public on(event: 'commandInhibited', listener: (msg: CommandMessage, command: Command, response: string|Error) => void): this;
+		public on(event: 'commandRun', listener: (msg: CommandMessage, command: Command, params: any[], response: any) => void): this;
+		public on(event: 'commandUnknown', listener: (msg: DiscordMessage, command: string) => void): this;
```

Comment:
`msg: ExtendedMessage`. It's not a DiscordMessage, but a much extended one.

Reply:
Right (My bad)